src/Compilers/CSharp/Portable/Parser/DocumentationCommentParser.cs

```diff
@@ -288,6 +288,16 @@ private XmlNodeSyntax ParseXmlElement()
                 _pool.Free(attrs);
             }
         }
+
+        private static bool MatchingXmlNames(XmlNameSyntax name, XmlNameSyntax endName)
+        {
+            // NOTE we do not compare colon tokens.
+            //      as long as name and prefix are matching we will consider names equivalent
+            //      it would be a different error if colon are missing
+            return name.LocalName.ValueText == endName.LocalName.ValueText &&
+                name.Prefix?.Prefix.ValueText == endName.Prefix?.Prefix.ValueText;
```

Comment:
First is optional syntax node ( "stuff : " )  , the second is not optional token ( "stuff" ) inside it. They are confusingly named the same.